phpunit/class-performance-test.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Server-side performance tests
+ *
+ * @package Gutenberg
+ */
+
+// To run these tests, set the RUN_SLOW_TESTS environment variable to a truthy
+// value.
+if ( getenv( 'RUN_SLOW_TESTS' ) ) {
+	class Performance_Test extends WP_UnitTestCase {
+		function test_parse_large_post() {
+			$start = microtime( true );
+			$start_mem = memory_get_usage();
+
+			$html = file_get_contents(
+				dirname( __FILE__ ) . '/fixtures/long-content.html'
+			);
+			$parser = new Gutenberg_PEG_Parser;
+			$result = $parser->parse( $html );
+
+			$time = microtime( true ) - $start;
+			$mem = memory_get_usage() - $start_mem;
+
+			error_log( '' );
+			error_log( 'Memory used (KB) : ' . round( $mem / 1024 ) );
+			error_log( 'Time (seconds)   : ' . ( round( $time * 100 ) / 100 ) );
```

Comment:
The `error_log` here may have an undesirable effect on the `phpunit` output, with the results being added inline with the progress when running all tests:

![image](https://user-images.githubusercontent.com/134745/27971797-d757d1d2-6308-11e7-94ec-0b52646ef5b8.png)